src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -265,6 +265,41 @@ public void TestStartOnUnixWithBadPermissions()
             Assert.NotEqual(0, e.NativeErrorCode);
         }
 
+        [Theory]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapNotUapAot, "Non applicable for uap - RemoteInvoke works differently")]
```

Comment:
Is this a duplication of the test in processtests.cs? Those tests are run on all platforms, why duplicate the test here again?

Reply:
Yes, I was under the impression those weren't running on Unix - I'm very surprised none of them failed in the PR before, then. For example, https://github.com/dotnet/corefx/pull/25866/files#diff-5bc7793999b3bb4572fb50d828365f4fR281 should definitely have failed with the previous state of this change